Source/WebGPU/WebGPU/BindGroup.mm

```diff
@@ -78,13 +78,280 @@ static MTLRenderStages metalRenderStage(ShaderStage shaderStage)
         simd::make_float3(+1.79968f, -0.53503f, +0.00012f),
         simd::make_float3(-0.97284f, 0.30145f, -1.13348f));
 }
+
+static MTLPixelFormat metalPixelFormat(CVPixelBufferRef pixelBuffer, size_t plane)
+{
+    auto pixelFormat = CVPixelBufferGetPixelFormatType(pixelBuffer);
+    auto biplanarFormat = ^(int plane) {
```

Comment:
Can we use C++ lambdas instead of blocks?

Blocks are useful for passing between framework boundaries, but if we're entirely within a single framework, we should prefer C++ lambas

Reply:
I can make it a lambda, but what is the difference between C++ lambdas and blocks from clang's perspective? I thought they were basically the same.

Reply:
Oh, the difference is just being more idiomatic, rather than performance or anything like that.

Reply:
Sounds good, I'll switch it to a lambda 👍